libnavigation-core/src/main/java/com/mapbox/navigation/core/replay/history/ReplayEventSimulator.kt

```diff
@@ -120,6 +129,9 @@ internal class ReplayEventSimulator(
         // The frequency that replay updates will be broad-casted
         private const val replayUpdateSpeedMillis = 100L
 
+        // When there are no events to play, delay the coroutine
+        private const val isDonePlayingEventsDelayMillis = 1000L
```

Comment:
```suggestion
        private const val IS_DONE_PLAYING_EVENTS_DELAY_MILLIS = 1000L
```

Reply:
this commit is not yelling i swear
https://github.com/mapbox/mapbox-navigation-android/pull/2885/commits/b6d0b74b093aa6b04a59b708c3390639f74007e2